Patch imported from https://github.com/armbian/build.git at revision c55f6ea8bc220bded6695fdae8b0c5bfa891b688

Upstream-Status: Inappropriate [not author]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 drivers/net/wireless/rockchip_wlan/Kconfig  | 1 -
 drivers/net/wireless/rockchip_wlan/Makefile | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/net/wireless/rockchip_wlan/Kconfig b/drivers/net/wireless/rockchip_wlan/Kconfig
index 6bfc72b..d1cf9f5 100644
--- a/drivers/net/wireless/rockchip_wlan/Kconfig
+++ b/drivers/net/wireless/rockchip_wlan/Kconfig
@@ -35,7 +35,6 @@ menuconfig RTL_WIRELESS_SOLUTION
     bool "Realtek Wireless Device Driver Support"
     default y
 
-source "drivers/net/wireless/rockchip_wlan/rtl8188eu/Kconfig"
 source "drivers/net/wireless/rockchip_wlan/rtl8188fu/Kconfig"
 source "drivers/net/wireless/rockchip_wlan/rtl8189es/Kconfig"
 source "drivers/net/wireless/rockchip_wlan/rtl8189fs/Kconfig"
diff --git a/drivers/net/wireless/rockchip_wlan/Makefile b/drivers/net/wireless/rockchip_wlan/Makefile
index fb0e47b..10173fe 100644
--- a/drivers/net/wireless/rockchip_wlan/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/Makefile
@@ -1,6 +1,5 @@
 # SPDX-License-Identifier: GPL-2.0
 obj-$(CONFIG_AP6XXX)	+= rkwifi/
-obj-$(CONFIG_RTL8188EU)	+= rtl8188eu/
 obj-$(CONFIG_RTL8188FU)	+= rtl8188fu/
 obj-$(CONFIG_RTL8189ES) += rtl8189es/
 obj-$(CONFIG_RTL8189FS) += rtl8189fs/
-- 
2.7.4

